index.html

```diff
@@ -21,8 +21,7 @@
       <th>6pm</th>
       <th>Total</th>
     </tr>
-      <td>Downtown</td>
-      <td id = "inputs"></td>
+
 
 
 
```

Comment:
First thing to note, which you're probably aware of already, is that the file path in your script tag is missing an extension. That should have a *.js extension, and you file should be appropriately named as well.